src/o365/yammer/commands/message/message-get.ts

```diff
@@ -0,0 +1,112 @@
+import commands from '../../commands';
+import GlobalOptions from '../../../../GlobalOptions';
+import {
+  CommandOption, CommandValidate
+} from '../../../../Command';
+import YammerCommand from "../../../base/YammerCommand";
+import request from '../../../../request';
+
+const vorpal: Vorpal = require('../../../../vorpal-init');
+
+interface CommandArgs {
+  options: Options;
+}
+
+interface Options extends GlobalOptions {
+  id: number;
+}
+
+class YammerMessageGetCommand extends YammerCommand {
+  public get name(): string {
+    return `${commands.YAMMER_MESSAGE_GET}`;
+  }
+
+  public get description(): string {
+    return 'Returns a Yammer message';
+  }
+
+  public getTelemetryProperties(args: CommandArgs): any {
+    const telemetryProps: any = super.getTelemetryProperties(args);
+    telemetryProps.id = typeof args.options.id !== undefined;
+    return telemetryProps;
+  }
+
+  public commandAction(cmd: CommandInstance, args: CommandArgs, cb: () => void): void {
+    const requestOptions: any = {
+      url: `${this.resource}/v1/messages/${args.options.id}.json`,
+      headers: {
+        accept: 'application/json;odata.metadata=none',
+        'content-type': 'application/json;odata=nometadata'
+      },
+      json: true
+    };
+
+    request
+      .get(requestOptions)
+      .then((res: any): void => {
+        if (args.options.output === 'json') {
+          cmd.log(res);
+        }
+        else {
+          cmd.log({
+            id: res.id, 
+            sender_id: res.sender_id, 
+            replied_to_id: res.replied_to_id, 
+            thread_id: res.thread_id, 
+            group_id: res.group_id,
+            created_at: res.created_at,
+            direct_message: res.direct_message,   
+            system_message: res.system_message,
+            privacy: res.privacy,
+            message_type: res.message_type,
+            content_excerpt: res.content_excerpt
+          });
+        }
+        cb();
+      }, (err: any): void => this.handleRejectedODataJsonPromise(err, cmd, cb));
+  }
+
+  public options(): CommandOption[] {
+    const options: CommandOption[] = [
+      {
+        option: '--id',
```

Comment:
Please do change  it to `option: '--id <id>',`. If `<id>` not at the end vorpal thinks it is a boolean, not a number.